analytics_data_api/v0/serializers.py

```diff
@@ -49,6 +49,26 @@ class ProblemSubmissionCountSerializer(serializers.Serializer):
     correct = serializers.IntegerField(default=0)
 
 
+class ConsolidatedAnswerDistributionSerializer(serializers.Serializer):
```

Comment:
I've now made it a subclass of ProblemResponseAnswerDistributionSerializer, which makes its use a little more obvious but makes the declaration significantly more verbose.  Not totally sure which is better, but I think it's okay as a subclass.